src/renderer/components/ObservationCard.tsx

```diff
@@ -116,6 +178,7 @@ function ObservationCard(props: ObservationCardProps) {
             {predictionsTable}
             {photoDetails}
           </div>
+          {isMaximized && navigation}
```

Comment:
When the two versions of the component start getting more and more different, we end up needing more and more conditional code like here. This increases the [cyclomatic complexity](https://en.wikipedia.org/wiki/Cyclomatic_complexity) of the code and makes it harder to understand and maintain.

I guess you wanted to use a single component for maximized and non-maximized cards to avoid code duplication. This makes a lot of sense :slightly_smiling_face: Hopefully there are other ways to reuse code. One option would be to have a helper component which could be used in both versions.

I think we'd gain quite a lot by refactoring this code to use separate components for maximized and non-maximized cards. It could be in this PR or another one. LMK if you'd like to discuss it :smiley:

Reply:
Sure, thx. I'll refactor it in this PR